frontend/components/InteractiveInputs/InteractiveInputs.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import ImageSlider from "../InteractiveImage/ImageSlider";
+import HolonButton from "../VersionOne/Buttons/HolonButton";
+import s from "./InteractiveInputs.module.css";
+
+type Props = {
+  input: {
```

Comment:
If the component only takes one prop called `input`, do you think it would be better if we instead got rid of `input` and did this?

```tsx
type Props = {
  id?: number;
  name?: string;
  type?: string;
  animationTag?: string;
  options: InteractiveInputOptions[];
}
```

... and then have the caller do `<InteractiveInputs {...ct.value} />`. This makes the API for the component slightly nicer when testing or using it elsewhere in the app.

Reply:
@antw, 
I understand the input-level is obsolete, so I have changed the code, but I nam not completely certain if this is what you mean ,Could you take another look at it?

Reply:
Yep, what you've done is exactly what I had in mind. :+1: